src/types/Todo.ts

```diff
@@ -0,0 +1,12 @@
+export interface Todo {
+  id: number;
+  userId: number | null;
+  title: string;
+  completed: boolean;
+}
+
+export interface TodoToSend {
+  userId: number | null;
+  title: string;
+  completed: boolean;
+}
```

Comment:
you can use Omit to avoid duplication
https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys